README.md

```diff
@@ -1931,6 +1931,181 @@ parent       <uuid>
  ```
 
 
+### **Commands V2**
+Commands V2.0 command is used to list the command requests, statuses, fire different actions on devices or groups like ping, install application etc. It provides advanced device commands capabilities like queuing, support for offline devices, dynamic device set for commands and command history.
+
+```sh
+$ espercli commandsV2 [SUB-COMMANDS]
+```
+#### Sub commands
+#### 1. list
+List and filter command requests.
+```sh
+$ espercli commandsV2 list [OPTIONS]
+```
+##### Options
+| Name, shorthand     | Default| Description|
+| ------------------- |:------:|:----------|
+| --commandtype, -ct  |        | Filter by type of command request |
+| --device, -d        |        | Filter by device name |
+| --devicetype, -dt   |        | Filter by device type |
+| --command, -c       |        | Filter by command name |
+| --issuedby, -i      |        | Filter by user id |
+| --limit, -l         | 10     | Number of results to return |
+| --json, -j          |        | Render result in JSON format |
+
+##### Example
+```sh
+$ espercli commandsV2 list -ct device -dt active -d DEV-ELOP-W57Z -l 2
+
+Total Number of Command Requests: 36
+
+RQUEST ID                             COMMAND               STATUS                                      ISSUED BY
+242f75a0-ea56-4cb6-898b-c017eaa01758  SET_KIOSK_APP       [{'state': 'Command TimeOut', 'total': 1}]  {'id': 12491, 'username': 'alok'}
```

Comment:
Can we avoid the json? Like just put Command TimeOut in status and alok in issued by?

Reply:
I had tried it for issued_by. But according to the spec, it is returned as a string and not json. So couldn't extract it. I can maybe do a spilt on the string and get the required response.

Reply:
@BindyaBhaskar  you can parse string into json and then extract it.